### DISCUSSION TITLE
Updates to this repo are on their way!!

### DISCUSSION THREAD

**Discussion by Xtendera** [varszegimarcell/Optiplex-3020-Hackintosh-OpenCore#62] [macOS 12 Monterey]
And no, I didn't use -no_compat_check.
![Mt12](https://user-images.githubusercontent.com/67883305/139540100-80b7ac3e-ce3c-44a9-ba02-30e105e88be7.png)

**Comment by JohnAker33** [comment]
Does it look like iMac 16,2 is the best SMBIOS choice ?

**Reply by Xtendera** [reply]
Yeah, probably. Iris graphics 6200 is the most similar to 4600. HD 6000 is very different compared to 4600 in terms of the way the OS communicates with it. I have done some performance testing with Cinebench and I seem to be getting average scores. I am currently testing DirectX and seeing its performance. I haven't found anything not working yet. Sleeping, audio jack, and USB seem to work fine. But I'll keep testing it out to find any issues.

**Reply by mmfixemer** [reply]
just a small question im running the current EFi setup with an dedicated nvidia Geforce GT 710 2gb model in dual monitor and I disabled the intel card itself. Do you think this setup still works with Monterey (since i read somewhere that Nvidia support Has Been Dropped
![Screen Shot 2022-03-19 at 9 15 45 PM](https://user-images.githubusercontent.com/80593326/159137092-06e73c3f-f73d-4f98-bebe-f55c95804e5d.png)
)

**Comment by varszegimarcell** [comment]
Hello, sorry that I had long gone from this repo.

I finished university, and I had a job, and sometimes due to that, I have to travel a lot abroad. 

On this weekend, I will have some freetime, and I will update the repo regarding to masOS 12, as well as OC, as well as the guide. 

I just startetd digging OpenCore's Monetrey guide, and it reccomends for Haswell the SMBIOS iMac17,1 for some reason... I will try to figure out which one is better, probably the 17,1 will keep working longer than the 16,2. Due to that, I have to regenerate my serials to get a working iMessage again, but that's an antoher stroy.

I will try my best on the weekend ;)

**Reply by Xtendera** [reply]
Hello,

Welcome back!! We have certainly missed you. I have also been just as busy. It seems like whenever I sit down to update this repo, something goes wrong. First my hard drive dies. Then I can't even push commits to the repo. Anyways, I have been stuck in homework, and since my drive did die, my progress in updating the repo went with it. I recently contracted COVID which is a nice touch. So yeah, seems like it has been a long year. I will also try to pick up work on the reformatted guide branch if I can. My hackintosh seems to be getting used less and less.

And by the way, I totally get why you could not update the repo. It seems like the world is running out of time. I got more motivation now, so I will actually try to work on this and some of my other projects.

Thank you,
Ashray (this is actually my real name this time. I have changed my name on github 3 times. 😆  )
